### ISSUE TITLE
lookup_passwordstore integration test fails on EL7

### ISSUE THREAD

**Issue by jpmahowald** [ansible/ansible#59531]
##### SUMMARY
lookup_passwordstore integration test fails on EL7

##### ISSUE TYPE
- Bug Report

##### COMPONENT NAME
lib/ansible/plugins/lookup/passwordstore.py

##### ANSIBLE VERSION
```
ansible 2.9.0.dev0
  config file = None
  configured module search path = [u'/home/John/.ansible/plugins/modules', u'/usr/share/ansible/plugins/modules']
  ansible python module location = /home/John/source/ansible/lib/ansible
  executable location = /home/John/source/ansible/bin/ansible
  python version = 2.7.5 (default, Jun 20 2019, 20:27:34) [GCC 4.8.5 20150623 (Red Hat 4.8.5-36)]
```

##### CONFIGURATION
```

```

##### OS / ENVIRONMENT
CentOS 7 (GCP image)
```
yum install python-tox python-virtualenv python2-crypto python2-junit_xml  libselinux-python gcc libyaml-devel 
```

##### STEPS TO REPRODUCE
```
ansible-test integration --python 2.7 --tox --allow-destructive  --allow-root --allow-unsupported lookup_passwordstore
```

##### EXPECTED RESULTS
Test without error

##### ACTUAL RESULTS
pass package is not available.
```
TASK [lookup_passwordstore : Install package] ************************************************************
fatal: [testhost]: FAILED! => {"changed": false, "msg": "No package matching 'pass' found available, installed or updated", "rc": 126, "results": ["No package matching 'pass' found available, installed or updated"]}
```

**Comment by jpmahowald** [comment]
EPEL 7's package was removed as broken. https://bugzilla.redhat.com/show_bug.cgi?id=1639806   Apparently there is an incompatibility with its use of git and tree. Consider finding some other repo with the right version. Or digging through EPEL for an old without fixes pass version.  https://bodhi.fedoraproject.org/updates/?packages=pass&release=epel-7

**Comment by samdoran** [comment]
This was "fixed" in 4291f09a68e60b41339f0597bb36a2e72fa94241 by skipping this test on EL since the package is no longer available.
